graph/model/user.go

```diff
@@ -0,0 +1,7 @@
+package model
+
+type User struct {
+	Username string `json:"username" validate:"required"`
+	Email    string `json:"email"  validate:"email"`
+	Password string `json:"password" validate:"min=8,max=32,alphanum"`
```

Comment:
Doesn't have to be added now, but we will need `About` and `Admin` here too eventually. (In reference to [this message in the Discord](https://discord.com/channels/856595632030613524/863809202145067019/867653627015725066).